src/fragments/lib-v1/graphqlapi/native_common/advanced-workflows/common.mdx

```diff
@@ -138,9 +138,21 @@ import flutter5 from '/src/fragments/lib-v1/graphqlapi/flutter/advanced-workflow
 
 <Fragments fragments={{ flutter: flutter5 }} />
 
+<InlineFilter filters={['swift', 'android']} >
 ## Combining Multiple Operations
+</InlineFilter>
 
+<InlineFilter filters={['flutter']} >
+## Combining multiple GraphQL operations in a single request
+</InlineFilter>
```

Comment:
I don't understand this, why we need a different header here based on platforms. Just rename it all to 
"## Combining multiple GraphQL operations in a single request"